src/edu/washington/escience/myria/CsvTupleWriter.java

```diff
@@ -57,9 +62,20 @@ public void writeColumnHeaders(final List<String> columnNames) throws IOExceptio
   public void writeTuples(final ReadableTable tuples) throws IOException {
     final String[] row = new String[tuples.numColumns()];
     /* Serialize every row into the output stream. */
+    Schema tbsc = tuples.getSchema();
     for (int i = 0; i < tuples.numTuples(); ++i) {
       for (int j = 0; j < tuples.numColumns(); ++j) {
-        row[j] = tuples.getObject(j, i).toString();
+        Type type = tbsc.getColumnType(j);
+        if (type.equals(Type.BLOB_TYPE)) {
+          // write the file out
+          // add filename to the csv file
+          String filename = UUID.randomUUID().toString();
```

Comment:
Yes, i need the name for the  csv file. what is the advantage of using File.createTempFile() rather than doing what I am doing?